host/sgx/create.c

```diff
@@ -1061,6 +1101,9 @@ oe_result_t oe_terminate_enclave(oe_enclave_t* enclave)
         free(enclave->debug_enclave);
     }
 
+    /* Destroy the ecall id table */
+    free(enclave->ecall_id_table);
```

Comment:
Should we check if pointer is NULL? Not sure if that's even possible tho.

Reply:
Fixed.